net-wireless/wavemon/wavemon-0.9.1.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools linux-info toolchain-funcs
+
+DESCRIPTION="Ncurses based monitor for IEEE 802.11 wireless LAN cards"
+HOMEPAGE="https://github.com/uoaerg/wavemon/"
+SRC_URI="https://github.com/uoaerg/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~hppa ~ppc ~sparc ~x86"
+
+IUSE="caps"
+RDEPEND="dev-libs/libnl:3
+	sys-libs/ncurses:0=
+	caps? ( sys-libs/libcap )"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
```

Comment:
this should be in BDEPEND instead

https://devmanual.gentoo.org/general-concepts/dependencies/index.html

Reply:
I moved it to `BDEPEND` on the 0.9.2 ebuild (as I removed 0.9.1 as stated in the other thread)